x-pack/plugins/cases/common/types/api/case/v1.ts

```diff
@@ -168,11 +168,11 @@ export const CasesFindRequestRt = rt.intersection([
       /**
        * The status of the case (open, closed, in-progress)
        */
-      status: CaseStatusRt,
+      status: rt.union([CaseStatusRt, rt.array(CaseStatusRt)]),
       /**
        * The severity of the case
        */
-      severity: CaseSeverityRt,
+      severity: rt.union([CaseSeverityRt, rt.array(CaseSeverityRt)]),
```

Comment:
What would be the risk of not doing it?

Reply:
I don't think there is any point in allowing duplication of values and this would be an easy way to cover a subset of scenarios where that could happen. (`severity.length` being bigger than the number of distinct values)

Reply:
We agreed on not applying this now but take it into account in coming prs